Files/Views/YourHome.xaml.cs

```diff
@@ -85,6 +85,18 @@ private async void RecentFilesWidget_RecentFileInvoked(object sender, UserContro
                             return;
                         }
                     }
+                    foreach (DriveItem drive in AppSettings.CloudDrivesManager.Drives)
```

Comment:
Since the two for loops look identical we can just loop on the two lists together

Reply:
One is looping over DrivesManager, and the other CloudDrivesManager.